app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -143,6 +144,14 @@ class AttendeeViewModel(
         )
     }
 
+    fun updateTicketDetailsVisibility() {
+        if (ticketDetailsVisibility.value == null) {
+            ticketDetailsVisibility.value = false
+        } else {
+            ticketDetailsVisibility.value = !ticketDetailsVisibility.value!!
```

Comment:
Please do not use non-null assertion.

Reply:
I will fix this. And some of my PRs keep having build fail because UndocumentedPublicFunction error, so how should I avoid this or does that mean I have to write some documentations/comments for that ?

Reply:
I think build will not fail for UndocumentedPublicFunction. Codacy will fail, which is okay.